source/org/zfin/framework/api/FilterFunction.java

```diff
@@ -42,4 +42,7 @@ static boolean fullMatchMultiValueOR(String entity, String value, String delimit
     }
 
 
+    static boolean valueIsNull(List list){
```

Comment:
what does this method do?

Reply:
this was meant to filter out those cases where no collection is found. Currently, there is no usage but will come in the near future.